### DISCUSSION TITLE
Welcome to WebPress Discussions!

### DISCUSSION THREAD

**Discussion by XHiddenProjects** [XHiddenProjects/WebPress#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aukc1970** [comment]
Good afternoon I wanted to install the CMS locally on the local Laragon server, but nothing is learned, an error appears Warning: Invalid argument supplied for foreach() in C:\laragon\www\webpress\WebPress\libs\users.lib.php on line 339

Fatal error: Uncaught TypeError: Return value of Users::ListUsers() must be of the type array, null returned in C:\laragon\www\webpress\WebPress\libs\users.lib.php:342 Stack trace: #0 C:\laragon\www\webpress\WebPress\plugins\Core\Core.plg.php(23): Users::ListUsers(false) https://github.com/XHiddenProjects1/WebPress/discussions/1 C:\laragon\www\webpress\WebPress\libs\plugin.lib. php(19): Core_head(NULL) https://github.com/XHiddenProjects1/WebPress/discussions/2 C:\laragon\www\webpress\WebPress\libs\plugin.lib.php(29): Plugin::myHook('head', 'Core', NULL) https://github.com/XHiddenProjects1/WebPress/issues/3 C:\laragon\www\webpress\WebPress\header.php(207): Plugin::hook('head') https://github.com/XHiddenProjects1/WebPress/pull/4 C:\laragon\www\webpress\WebPress\auth.php(14): head(' Register', '..') https://github.com/XHiddenProjects1/WebPress/pull/5 {main} thrown in C:\laragon\www\webpress\WebPress\libs\users.lib.php on line 342

Please help me figure out why SMS is not installed

**Reply by XHiddenProjects** [reply]
Sorry for not getting back to you sooner, but I will continue this project and fix everything up. I wasn't sure if would like to continue this, but I will. Just give me time to clean everything up.
